DESCRIPTION

```diff
@@ -1,48 +1,26 @@
 Package: hubEnsembles
-Title: Ensemble methods for combining hub model outputs
+Title: Ensemble Methods for Combining Hub Model Outputs
 Version: 0.1.6
 Authors@R: c(
-    person(given = "Evan L",
-           family = "Ray",
-           email = "elray@umass.edu",
-           role = c("aut")),
-    person(given = "Li",
-           family = "Shandross",
-           role = c("aut", "cre"),
-           email = "lshandross@umass.edu",
+    person("Evan L", "Ray", , "elray@umass.edu", role = "aut"),
+    person("Li", "Shandross", , "lshandross@umass.edu", role = c("aut", "cre"),
            comment = c(ORCID = "0009-0008-1348-1954")),
-    person(given = "Emily",
-           family = "Howerton",
-           role = c("aut"),
-           email = "ehowerton@psu.edu",
+    person("Emily", "Howerton", , "ehowerton@psu.edu", role = "aut",
            comment = c(ORCID = "0000-0002-0639-3728")),
-    person("Anna", "Krystalli", , "annakrystalli@googlemail.com", 
-           role = c("ctb"),
+    person("Anna", "Krystalli", , "annakrystalli@googlemail.com", role = "ctb",
            comment = c(ORCID = "0000-0002-2378-4915")),
-    person("Nicholas G.", "Reich", email = "nick@umass.edu",
-           role = c("ctb"),
-           comment = c(ORCID = "0000-0003-3503-9899")))
+    person("Nicholas G.", "Reich", , "nick@umass.edu", role = "ctb",
+           comment = c(ORCID = "0000-0003-3503-9899"))
+  )
 Description: Functions for combining model outputs (e.g. predictions or
-    estimates) from multiple models into an aggregated ensemble model output.
+    estimates) from multiple models into an aggregated ensemble model
+    output.
 License: MIT + file LICENSE
-VignetteBuilder: knitr
-Suggests: 
-    cowplot,
-    ggplot2,
-    hubData,
-    hubExamples,
-    hubVis (>= 0.0.0.9100),
-    knitr,
-    rmarkdown,
-    scales,
-    testthat (>= 3.0.0)
-Config/testthat/edition: 3
-Encoding: UTF-8
-Roxygen: list(markdown = TRUE)
-RoxygenNote: 7.3.1
 URL: https://github.com/hubverse-org/hubEnsembles,
     https://hubverse-org.github.io/hubEnsembles/
 BugReports: https://github.com/hubverse-org/hubEnsembles/issues
+Depends: 
+    R (>= 2.10)
```

Comment:
This version of R is the lowest possible version because the base pipe (`|>`) is used in the package and, unfortunately, it's not possible to backport. That being said, because this package uses tidyverse packages, the minimum version of R they support right now is 4.0 and next year, it will be 4.1: https://www.tidyverse.org/blog/2019/04/r-version-support/

```suggestion
    R (>= 4.10)
```